crates/air-lib/interpreter-interface/src/call_service_result.rs

```diff
@@ -16,17 +16,20 @@
 
 use air_interpreter_sede::define_simple_representation;
 use air_interpreter_sede::derive_serialized_type;
-use air_interpreter_sede::JsonFormat;
+use air_interpreter_sede::MsgPackMultiformat;
 use air_interpreter_sede::Representation;
 use serde::Deserialize;
 use serde::Serialize;
 use serde_json::Value as JValue;
 use std::collections::HashMap;
 
-pub type CallResults = HashMap<u32, CallServiceResult>;
+/// This is a map from a String to a service result for compatibility with JavaScript.
+/// Binary format implementations like `rmp-serde` do not bother converting keys from strings, unlike `serde_json`.
```

Comment:
looks like it's obsolete

Reply:
No, it explains why `String` is used at the next line.